lib/guardian_db/token.ex

```diff
@@ -5,19 +5,15 @@ defmodule GuardianDb.Token do
 
   use Ecto.Schema
   import Ecto.Changeset
-  import Ecto.Query, only: [from: 2]
+  import Ecto.Query, only: [where: 2]
 
+  alias Ecto.Query
   alias GuardianDb.Token
 
-  config = Application.get_env(:guardian_db, GuardianDb, [])
-
   @primary_key {:jti, :string, autogenerate: false}
-  @schema_name Keyword.get(config, :schema_name, "guardian_tokens")
-  @schema_prefix Keyword.get(config, :prefix)
-
   @allowed_fields ~w(jti typ aud iss sub exp jwt claims)a
 
-  schema @schema_name do
+  schema "virtual: token" do
```

Comment:
What is the thinking behind this change?

Reply:
@doomspork related to https://github.com/ueberauth/guardian_db/issues/55